robot/control/modules/motion-control/MotionControl.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <chrono>
+
+#include "RobotModel.hpp"
+#include "RobotEstimator.hpp"
+#include "RobotController.hpp"
+
+namespace motion {
+
+using clock = std::chrono::steady_clock;
+using time_point = std::chrono::time_point<clock>;
```

Comment:
Normally I'd agree with you, but it's within the `motion` namespace so it doesn't pollute the global namespace which is a case I think it's reasonable to do it for clarity. The other, more specific reason I used it here is to abstract the the clock away from the places that use it to force people who use this later to use `steady_clock` instead of `system_clock` (it doesn't really matter on these robots because we probably are never going to add timezone support in, but regardless it's probably better practice).

Reply:
I'm down, that sounds like a good enough reason to me